### PR TITLE
[arabic_izza] Add files via upload

### PR DESCRIPTION
**Pull request by MaleBou**
Files updated:
kmn file
Touch layout ( tablet and phone).
Help

### CODE REVIEW

File: Izza3-layout.js [hunk 1]
```diff
@@ -0,0 +1,1172 @@
+{
```
Comment (darcywong00) [inline Izza3-layout.js]: Can you move these changes to `release\a\arabic_izza\source\arabic_izza-layout.js`?

File: arabic_izza-2017.01.js [hunk 2]
```diff
@@ -0,0 +1,1548 @@
+KeymanWeb.KR(new Keyboard_arabic_izza());
```
Comment (darcywong00) [inline arabic_izza-2017.01.js]: You can remove this file from the PR because the Keyman build server will rebuild it (`arabic_izza.js`)

**Comment by MaleBou** [comment]
I am sorry for not being able to reply sooner because I was away in an international conference. I will try to move the files as requested.

    On Wednesday, April 18, 2018, 2:07:33 AM GMT+1, Darcy Wong <notifications@github.com> wrote:  
 
 
@darcywong00 requested changes on this pull request.

@MaleBou

Thank you for this submitting these files. Since the current source is located at release\a\arabic_izza\source, some of these changes need to be moved there.

Could you also update release\a\arabic_izza\HISTORY.md with a brief summary of your changes?

Let @DavidLRowe know if you would like some help moving these files with Git.

In Arabic Izza.kmn:
> @@ -0,0 +1,135 @@
+﻿c Arabic Izza is the result of about forty years of researching Arabic language and script by Prof.  Abdelmalek Bouhadjera.

This entire file needs to moved to release\a\arabic_izza\source\arabic_izza.kmn

In Izza3-help.htm:
> +Shadda under letter Shin for example, using Shift.
+الحركات و الشدة لها أزرار مستقلة على المستوى الأول
+توجد كذلك على المستوى الثاني تحت الحروف الأولى لأسمائها
+على المستوى الثاني توجد كذلك كل الرموز لكتابة العربية بدقة و احترافية
+
+Touch Layouts
+Tablet: the layout remains unchanged.
+التصميم كما في حواسيب المكتب 
+
+Phone: beside taking advantage of the main points above, the number of keys is reduced to eight 
+in each line on the first layer, thus improving enormously the typing speed.
+إلى جانب العمل بالنقاط المذكورة أعلاه، لقد تم تخفيض عدد المفاتيح في كل سطر إلى ثمانية على المستوى
+الأول، مما يضاعف من سرعة إدخال النصوص العربية
+ 
+© 1977-2018 Prof. Abdelmalek Bouhadjera
+

Could you apply these changes to release\a\arabic_izza\source\help\arabic_izza.php?

That way, when this PR gets approved and merged, the online help at https://help.keyman.com/keyboard/arabic_izza gets automatically updated.

In Izza3-layout.js:
> @@ -0,0 +1,1172 @@
+{

Can you move these changes to release\a\arabic_izza\source\arabic_izza-layout.js?

In arabic_izza-2017.01.js:
> @@ -0,0 +1,1548 @@
+KeymanWeb.KR(new Keyboard_arabic_izza());

You can remove this file from the PR because the Keyman build server will rebuild it (arabic_izza.js)

**Comment by MaleBou** [comment]
New files have been loaded

File: Arabic Izza.kmn [hunk 3]
```diff
@@ -0,0 +1,135 @@
+﻿c Arabic Izza is the result of about forty years of researching Arabic language and script by Prof.  Abdelmalek Bouhadjera.
```
[the remark this answers is not in the document]
Comment (MaleBou) [reply Arabic Izza.kmn]: I have loaded the three files to Malebou-Izza branch.

**Comment by darcywong00** [comment]
You also have a different PR #240 for arabic_izza updates. Are you intending to continue with this PR or that one?

Both of your arabic_izza PR's are based on the older directory structure of the keyboards repo, so you'll need to sync your master branch and rebase your work. (see my comments on #240)

**Comment by MaleBou** [comment]
Let's continue with PR #240 for now.
I am experimenting with GIT GUI for the moment.

**Comment by MaleBou** [comment]
I have followed all the instructions for sync and nothing seems to change on my Master branch apart from the files related to the new PR.

**Pull request closed without merging**